src/main/java/in/projecteka/consentmanager/link/LinkConfiguration.java

```diff
@@ -46,14 +46,16 @@ public Link link(WebClient.Builder builder,
                      CentralRegistry centralRegistry,
                      GatewayServiceProperties gatewayServiceProperties,
                      LinkServiceProperties serviceProperties,
-                     CacheAdapter<String, String> linkResults) {
+                     CacheAdapter<String, String> linkResults,
+                     CacheAdapter<String, String> patientLinkReferenceResults) {
```

Comment:
Do you need another CacheAdapter here? You can reuse the same linkResults - right?

Reply:
done